### PR TITLE
feat: diff sync

### PR DESCRIPTION
**Pull request by abulte**
Permanent fix https://github.com/ecolabdata/ecospheres-dashboard-backend/issues/33

Prod

```
Feeding 0 datasets...
Removing 3 datasets...
<delete_topic_datasets: elapsed=0.00s>
Total count: 24584, elapsed: 169.85 s
```

Demo

```
Feeding 5 datasets...
<put_topic_datasets: count=5, elapsed=0.00s>
Removing 500 datasets...
<delete_topic_datasets: elapsed=0.00s>
Total count: 23238, elapsed: 212.53 s
```

### CODE REVIEW

File: feed-universe.py [hunk 1]
```diff
@@ -322,12 +327,20 @@ def check_sync(org: Organization, datasets: list):
...
             if args.check:
-                check_sync(org, datasets)
-                continue
-
-            print(f"Feeding {len(datasets)} datasets from '{org.slug}'...")
-            for batch in batched(datasets, 1000):
-                api.put_topic_datasets(topic, batch)
+                check_sync(topic_slug, org, datasets)
+            else:
+                new_datasets += datasets
```
Comment (streino) [inline feed-universe.py:332]: Since `new_datasets` is always initialized, let's always maintain it?

Comment (streino) [reply feed-universe.py:334]: Hopefully we remember after a few more refactors ;)
Would such an init work? Makes usage more explicit.
```
new_datasets = [] if not args.check else None
```

**Review by streino — APPROVED** [review]

File: feed-universe.py (hunk 1 above)
Comment (abulte) [reply feed-universe.py:334]: Nope, the typing system is not smart enough :-/

Comment (streino) [reply feed-universe.py:334]: :shrug:

**Pull request merged**